tests/jsx-significant-space/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -100,16 +101,14 @@ before_break2 = (
   <span>
     <span
       barbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbar
-    />
-    {" "}
+    />{" "}
     foo
   </span>
 );
 
 after_break = (
   <span>
-    foo
-    {" "}
+    foo{" "}
```

Comment:
I think the existing behavior was intentionally added by @vjeux (IIRC, it used to behave as you have it for jsx whitespace)